### DISCUSSION TITLE
Cannot create report from command line

### DISCUSSION THREAD

**Discussion by manjunath-ZEE** [WasiqB/ultra-reporter-app#50] [Q&A] [answered]
I have installed ultra-report using npm command. I am trying to integrate this with my existing java testNg framework. I want to genarate this report along with my execution. While trying getting the below error while same is working fine if the xml uploaded in UI.
 
  @WasiqB Please suggest is there any other way to integrate and use this reporting in existing frame work
 
 ultra-report target/surefire-reports/testng-results.xml

✔ XML File exists
✖️ Error while generating JSON from XML

Message: TypeError - Cannot read properties of undefined (reading 'ignored')

ℹ If you feel the issue is related to the report, please open an issue on GitHub 👇
🔗 https://github.com/WasiqB/ultra-report/issues/new

**Comment by WasiqB** [comment, marked as answer]
@manjunath-ZEE , Ultra-report CLI project is not being maintained anymore.

Please check if your file is getting generated with this projects Web application.
